src/components/Text/Text.tsx

```diff
@@ -42,7 +42,7 @@ export const useStyles = makeStyles(
       letterSpacing: theme.pxToRem(0.5),
     },
     tooltip: {
-      fontSize: theme.pxToRem(10),
+      fontSize: theme.pxToRem(11),
```

Comment:
@dexterca Not sure why we're using `pxToRem` here for Text demo and not `fontSizes.tooltip`? Any ideas?

Reply:
Yeah not sure, I'm good with going with fontSizes if you are.

Reply:
I'll make the `fontSizes` change in another PR since it affects all sizes in the Text demo